app/CPReflectionService/CPReflectionService.m

```diff
@@ -42,4 +42,21 @@ - (void)pluginsFromPodfile:(NSString * _Nonnull)contents
   }];
 }
 
+- (void)installedPlugins:(void (^ _Nonnull)(NSArray<NSString *> * _Nonnull plugins, NSError * _Nullable error))reply;
+{
+  [RBObject performBlock:^{
+
+    reply(@[@"cocoapods-try", @"cocoapods-trunk"], nil);
+
+//    RBObjectFromString(@"require 'claide'");
+//    RBPluginManager *pluginManager = RBObjectFromString(@"CLAide::Command::PluginManager");
+//    [pluginManager load_plugins:@"cocoapods"];
+//    NSArray *specs = [pluginManager specifications];
+//    reply([specs valueForKeyPath:@"name"], nil);
```

Comment:
See https://cocoapods.slack.com/archives/cocoapods-app/p1449110611000472

Or https://gist.github.com/orta/12ec2e74b9427d201006 for a full error log